pkg/registry/global_registry.go

```diff
@@ -0,0 +1,25 @@
+package registry
+
+import (
+	"fmt"
+
+	"github.com/vmware/go-ipfix/pkg/entities"
+)
+
+var globalReg map[uint32]map[uint16]entities.InfoElement
+
+func LoadRegistry() {
```

Comment:
Create an issue in Antrea to make the change for loading registry accordingly for new version of IPFIX library.

Reply:
Do you refer to the global registry refactoring?

Reply:
Yes global registry change. We need to make corresponding Antrea change if we move to next version of go-ipfix library.